File: src/main/java/com/example/hotel_mvc/model/entity/Company.java
package com.example.hotel_mvc.model.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.time.Instant;
import java.util.Date;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "company")
public class Company {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    private Integer id;

    @Column(name = "name", length = 45)
    private String name;

    @Column(name = "name_furigana", length = 45)
    private String nameFurigana;

    @Column(name = "phone_number", length = 45)
    private String phoneNumber;

    @Column(name = "email", length = 45)
    private String email;

    @Column(name = "prefCode", length = 45)
    private String prefCode;

    @Column(name = "address", length = 45)
    private String address;

    @Column(name = "postal_code", length = 45)
    private String postalCode;

    @Column(name = "building_name", length = 45)
    private String buildingName;

    @Column(name = "create_at")
    private Date createAt;

    @Column(name = "update_at")
    private Date updateAT;

}

File: src/main/resources/static/js/company.js
const t = new Date();
const date = ('0' + t.getDate()).slice(-2);
const month = ('0' + (t.getMonth() + 1)).slice(-2);
const year = t.getFullYear();


const time = `${date}/${month}/${year}`;
$(document).ready(function () {

    $('.table .eBtn').on('click', function (event) {
        event.preventDefault();
        var href = $(this).attr('href');
        $.get(href, function (lstcompany, status) {
            $('.myModal #id').val(lstcompany.id);
            $('.myModal #name').val(lstcompany.name);
            $('.myModal #name_furigana').val(lstcompany.nameFurigana);
            $('.myModal #phone_number').val(lstcompany.phoneNumber);
            $('.myModal #email').val(lstcompany.email);
            $('.myModal #pref_code').val(lstcompany.prefCode);
            $('.myModal #address').val(lstcompany.address);
            $('.myModal #postal_code').val(lstcompany.postalCode);
            $('.myModal #building_name').val(lstcompany.buildingName);
            $('.myModal #create_at').val(lstcompany.createAt);
            $('.myModal #update_at').val(lstcompany.updateAt);
        })
        $('.myModal #exampleModal').modal();
    });
});